File: Desenvolvimento/class/produto.php
<?php

class produto{
    private $codigo_produto;
    private $nome;
    private $cor_pred;
    private $espessura;

    public function setCodigo_produto($codigo_produto){
        $this->codigo_produto = $codigo_produto;
    }
    public function getCodigo_produto(){
        return $this->codigo_produto;
    }
    public function setNome($nome){
        $this->nome = $nome;
    }
    public function getNome(){
        return $this->nome;
    }
    public function setCor_pred($cor_pred){
        $this->cor_pred = $cor_pred;
    }
    public function getCor_pred(){
        return $this->cor_pred;
    }
    public function setEspessura($espessura){
        $this->espessura = $espessura;
    }
    public function getEspessura(){
        return $this->espessura;
    }
}
?>

File: Sistema de Corte de Rochas - Caso de Uso/src/javafx/domain/Produto.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package javafx.domain;

/**
 *
 * @author André
 */
public class Produto {

    private int codProduto;
    private String nome;
    private String corPred;
    private String espessura;

    public int getCodProduto() {
        return codProduto;
    }

    public void setCodProduto(int codProduto) {
        this.codProduto = codProduto;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getCorPred() {
        return corPred;
    }

    public void setCorPred(String corPred) {
        this.corPred = corPred;
    }

    public String getEspessura() {
        return espessura;
    }

    public void setEspessura(String espessura) {
        this.espessura = espessura;
    }

    @Override
    public String toString() {
        return nome;
    }

}
